lib/ansible/modules/storage/netapp/na_ontap_disks.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/python
+
+# (c) 2018, NetApp, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+
+module: na_ontap_disks
+
+short_description: NetApp ONTAP Assign and unassign disks from nodes
+extends_documentation_fragment:
+    - netapp.na_ontap
+version_added: '2.7'
+author: NetApp Ansible Team (ng-ansibleteam@netapp.com)
+
+description:
+- Assign all or part of disks to nodes.
+
+options:
+
+  state:
+    description:
+    - Whether the specified aggregate should exist or not.
+    choices: ['present', 'absent']
+    default: 'present'
+
+  node:
+    required: true
+    description:
+    - Used only with auto or all parameter. It specifies the node
```

Comment:
This doesn't make sense to me (though I'm not a NetApp user)
* `Used only with auto or all parameter` - What defines `auto` or `all`?
* Is this meant to be the name of a node? does node mean disk, cluster, ...?